resources/production/config-example.toml

```diff
@@ -138,7 +138,7 @@ qps_limit = 10
 address = '0.0.0.0:9999'
 
 # The number of event stream events to buffer.
-event_stream_buffer_length = 100
+event_stream_buffer_length = 10
```

Comment:
is this correct? i remember us agreeing to set the event streams qps to 10, but I don't remember deciding to reduce the number of events in the buffer.